composer.json

```diff
@@ -20,7 +20,9 @@
     "require": {
         "php": "^7.2",
         "cocur/slugify": "^1.4 || ^2.0 || ^3.0 || ^4.0",
-        "sonata-project/doctrine-extensions": "^1.1",
+        "sonata-project/exporter": "^1.11.0 || ^2.0",
+        "sonata-project/form-extensions": "^0.3",
+        "sonata-project/twig-extensions": "^0.2",
```

Comment:
Let's use 0.1-BETA for both here?

Reply:
Yes, also add ^3.0:
```
"jms/serializer-bundle": "^1.0 || ^2.0 **|| ^3.0**",
```

Some test need be fix. I will do it today.